geniza/corpus/migrations/0016_rename_probable_languages_to_secondary.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1 on 2021-08-12 15:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("corpus", "0015_add_document_date"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="document",
+            old_name="probable_languages",
+            new_name="secondary_languages",
+        ),
+        migrations.AlterField(
+            model_name="document",
+            name="languages",
+            field=models.ManyToManyField(
+                blank=True, to="corpus.LanguageScript", verbose_name="Primary Languages"
+            ),
+        ),
```

Comment:
do you know why one of these ended up as a `RenameField` and the other as an `AlterField`? is it because we only changed verbose name for `languages` but changed the actual field name for `secondary_languages`?

Reply:
@thatbudakguy correct